api/orchestrator/orchestrator.proto

```diff
@@ -39,6 +39,14 @@ option go_package = "clouditor.io/clouditor/api/orchestrator";
 
 // Manages the orchestration of components within the Clouditor architecture
 service Orchestrator {
+  // Get Metadata
+  rpc Metadata(MetadataRequest)
+  returns (MetadataResponse) {
+    option (google.api.http) = {
+      get: "/v1/orchestrator/metadata"
```

Comment:
maybe something related to "runtime" instead of metadata?

Reply:
For me ok.